python/py-bidi/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-bidi
+python.rootname     python-${python.rootname}
+version             0.4.2
+license             GNU
+maintainers         nomaintainer
+supported_archs     arm64 x86_64
```

Comment:
this doesn't make sense to me - does this package actually compile anything? To me it seems to be pure Python and therefore shouldn't it be `noarch`?

Reply:
Thanks done. (Copy-and-paste error.)